fileonfeature.txt

```diff
@@ -1,2 +1,3 @@
 this is my file on feature branch
 added some lines to edit the file 
+new line from new branch 
```

Comment:
this not the way how we add new lines to out txt files!
It should start with "added" for

Reply:
ok i'm goingt to change it.